src/goto-checker/bmc_util.h

```diff
@@ -188,7 +188,10 @@ void run_property_decider(
   "(unwind):" \
   "(unwindset):" \
   "(graphml-witness):" \
-  "(unwindset):"
```

Comment:
I was confused at first - maybe we should have a separate commit that cleans up this duplicate `(unwindset):`?